core/templates/dev/head/pages/story-editor-page/story-editor-page.mainpage.html

```diff
@@ -2,6 +2,12 @@
 <html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
   <head>
     @require('../../base_components/header.html', {"title": "Oppia"})
+    <script>
```

Comment:
Er ... @Jamesjay4199 @vojtechjelinek is this the intended method? I thought @Jamesjay4199 was trying to remove all the globals...

Reply:
I followed the pattern in topic-editor-page.mainpage.html. Maybe that hasn't been modified yet?

Reply:
`GCS_RESOURCE_BUCKET_NAME` is removed in an open PR, `INTERACTION_SPECS` will be removed in 3rd milestone because it is interconnected with `{{ interaction_templates }}`